include/mata/alphabet.hh

```diff
@@ -236,17 +238,17 @@ public:
      * Get the next value for a potential new symbol.
      * @return Next Symbol value.
      */
-    Symbol get_next_value() const { return next_symbol_value; }
+    Symbol get_next_value() const { return next_symbol_value_; }
 
     /**
      * Get the number of existing symbols, epsilon symbols excluded.
      * @return The number of symbols.
      */
-    size_t get_number_of_symbols() const { return m_symbols.size(); }
+    size_t get_number_of_symbols() const { return symbols_.size(); }
```

Comment:
rename to size()

Reply:
It was used earlier to express that we exclude epsilons, and return only alphabet symbols. Now, alphabets will know nothing about epsilons and this function can be therefore renamed to `size()` since it is not ambiguous any more.